.github/workflows/android-ci-pull.yml

```diff
@@ -28,14 +28,11 @@ concurrency:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Thank you! Is there a reason we are not using `ubuntu-latest`? Is this due to version pinning?

Reply:
Yes that is the reasoning.

Reply:
Dependabot can make pull requests to bump the versions used in github action workflows. Is it turned on at the moment for workflows?

Reply:
I may make sense to use Ubuntu 22 for the forseeable future because it is a LTS release.

Reply:
> Dependabot can make pull requests to bump the versions used in github action workflows. Is it turned on at the moment for workflows?

Yes, I think so.